### DISCUSSION TITLE
Chips are not working

### DISCUSSION THREAD

**Discussion by anantkj** [primefaces/community#403] [PrimeNG] [answered]
Currently we are facing issue after upgrade 16.7.1 version on system , chips are not working as expected event in prime ng documentation is not working regarding reactive forms chips so please check reactive forms with chips. it is disabled input after focus.

![Uploading primeng_issue.png…]()

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue clearly?

**Reply by anantkj** [reply]
https://primeng.org/chips#reactive-forms

Please check in original documentation with reactive forms not template driven , chips are not working here too.

it was working before , as i think in oct last week it was working fine but after recent changes in version it is not working

**Reply by cetincakiroglu** [reply, marked as answer]
Hi,

We've fixed this in 16.8.0, thanks a lot for the feedback and reporting the issue!

**Comment by cetincakiroglu** [comment]
Thanks for letting us know, we're working towards to fix it.

**Comment by cetincakiroglu** [comment]
Hi,

This issue is fixed and the fix will be available by upcoming release https://github.com/primefaces/primeng/issues/14062
